db/storage.go

```diff
@@ -158,6 +158,14 @@ func (s *Storage) Quota() *storage.Collection {
 	return c
 }
 
+// SAMLRequests returns the samlrequest from MongoDB.
```

Comment:
// SAMLRequests returns the saml_requests collections from MongoDB.
shoud be a better comment